Orange/data/table.py

```diff
@@ -204,11 +229,22 @@ def __new__(cls, *args, **kwargs):
         return cls.from_numpy(domain, *args, **kwargs)
 
     def __init__(self, *args, **kwargs):
-        # So subclasses can expect to call super without breakage; noop
-        pass
+        # TODO: use this instead of __new__
+        # all weights initialized to 1 (see the weight functions for details)
+        self[Table._WEIGHTS_COLUMN] = 1
+
+        self.name = kwargs.get("name") or self.name
```

Comment:
`name` [was set to `untitled`](https://github.com/biolab/orange3/pull/1347/files/f38001b4895ddeab11a0b1550f4e1dac97faf5dd#diff-1a46745833e0557e8e3cbff98c867258L148). It did kind of work. Was used, anyway.